scripts/custom-config.js

```diff
@@ -71,5 +73,18 @@ function rewireModule(modulePath, customizer) {
   // react-scripts imports the config as a `const` and we can't
   // modify that reference.
   let config = defaults.__get__('config');
+
+  // Have webpack compile the env keys so the can be used on the client
+  var env = dotenv.config().parsed;
```

Comment:
@edward-hong CRA has inbuilt configuration for .env file. If you have your keys renamed to REACT_APP_GA_TRACKING_ID in .env file, you can access the same via process.env.REACT_APP_GA_TRACKING_ID from js.

Reply:
Thanks for the heads up! I'll make sure to make the adjustments you suggested.